### DISCUSSION TITLE
change between song and video

### DISCUSSION THREAD

**Discussion by grumi123** [pear-devs/pear-desktop#764] [Ideas]
this app is excellent I think almost perfect, there are details such as not being able to switch between song and video as if it were premium there is no way to do it yet right?

**Comment by Araxeus** [comment]
there is a plugin called `video-toggle` which does exactly that

**Reply by grumi123** [reply]
really only lets you switch between the video thumbnail and the video itself, not like the premium option.

**Reply by Araxeus** [reply]
I don't see the difference, also you can use the `native` mode in the plugin option, which just enables the native premium toggle

**Reply by grumi123** [reply]
I mean when I play an album it won't queue the audio it will queue the video in case the song has it, that's supposed to be supplied by this plugin too, right?

**Reply by Araxeus** [reply]
I don't really understand what you are saying

**Reply by grumi123** [reply]
sorry my English is not so good, what I wanted to say was related to this, the article explains it better 
[https://support.google.com/youtubemusic/answer/6313574?hl=en&co=GENIE.Platform%3DAndroid](url)
